Server/src/lobbycoordinator.erl

```diff
@@ -73,6 +73,7 @@ loop(Rooms, Lobby_increment) ->
 	  
 	%This happens when you send a new diagram to the lobby
 	{Creator_Socket, {Did, Class_names, Classes, Messages}} -> 
+	  io:format("Im creating stuff ~n"),
```

Comment:
good for you

Reply:
Oh nice, you found it. Been looking for that

Reply:
+1